pages/[id].js

```diff
@@ -0,0 +1,5 @@
+import ActivityCardDetails from "@/components/ActivityCardDetails";
+
+export default function JokeDetailsPage() {
+  return <ActivityCardDetails />;
+}
```

Comment:
instead of creating a separate Details component, please move the content into this page component. this removes one level of nesting. Try to keep your component tree as shallow as possible as long as components size is still manageable.